setup_boostbook.py

```diff
@@ -11,7 +11,7 @@
 
 # User configuration
 # (MAINTANERS: please, keep in synch with setup_boostbook.sh)
-DOCBOOK_XSL_VERSION = "1.75.2"
+DOCBOOK_XSL_VERSION = "1.79.1"
```

Comment:
Version Version `1.79.2` gives errors in the chunking. The documentation is not correct anymore with the parameters supplied in current Jamfiles (at least, not for Boost.Geometry). It misses all tables of contents (apart from the one on the main page). Effectively it's completely broken.

That's likely the case with `1.79.1` as well.

Reply:
With version `1.79.1` , generation of Boost.Geometry is correct!